source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRViewController.mm

```diff
@@ -0,0 +1,127 @@
+//
+//  ACRViewController.m
+//  ACRViewController
+//
+//  Created by jwoo on 5/26/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRViewController.h"
+#import "ACRHostConfig.h"
+#import "AdaptiveCard.h"
+#import "TextBlock.h"
+#import "Image.h"
+#import <AVFoundation/AVFoundation.h>
+
+using namespace AdaptiveCards;
+
+@implementation ACRViewController
+std::shared_ptr<AdaptiveCard> adaptiveCards;
+ACRHostConfig* m_hostConfig;
```

Comment:
awesome, thanks

---
In reply to: [122549457](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122549457) [](ancestors = 122549457)